### DISCUSSION TITLE
How to add a new user to an App?

### DISCUSSION THREAD

**Discussion by cesarverdesslots** [Budibase/budibase#6525] [Help]
Sorry for this silly question...

Can you please reference to Budibase documentation where explains how to assign a user to an App?

For example this is the screen for my DEI application `Users` table:
![Screenshot from 2022-07-01 12-35-28](https://user-images.githubusercontent.com/83496099/176931928-5f6dd21c-382b-4ee6-924a-be06e084655c.png)

But as you can see, I had created another user (Daniel Lanfranchi) as Admin for the DEI application, but when I see the users attached to the app, only cesar.verdes@gruposlots.com.ar (how create the app) is shown 

![image](https://user-images.githubusercontent.com/83496099/176932383-968055cc-a58b-492d-bc4f-1f41fc8c28f2.png)

How can assign the user daniel.lanfranchi@gruposlots.com.ar in order to been shown in the `Users` table of the DEI application?

**Comment by melohagan** [comment]
HI @cesarverdesslots 

The docs page is here: https://docs.budibase.com/docs/adding-users
Although I don't think that adds to anything you've already tried.

Are you, cesar, an admin user for this app? 
If you add a non-admin user to the app do they appear?

**Comment by cesarverdesslots** [comment]
Dear @melohagan, I followed the documentation and I created a new user call Daniel Lanfranchi and give him Admin permission to the DEI app:

![image](https://user-images.githubusercontent.com/83496099/177320153-d17db7ba-60e5-46c2-9a15-3fc1b871adc7.png)

I'm the creator and administrator of the DEI app (César Verdes), but as you can see Daniel Lanfranchi user doesn't appear associated to de DEI app into the Users view::

![image](https://user-images.githubusercontent.com/83496099/177319226-7c73f08a-ba42-4268-b583-d8987ec861aa.png)

**Reply by melohagan** [reply]
Hi @cesarverdesslots Out of curiosity, is the app published?
